File: 800/Java/P11.java
import java.util.*;

public class P11 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        int mini = Integer.MAX_VALUE;
        for(int i = 0; i < n; i++) {
            int num = scanner.nextInt();
            mini = Math.min(mini, Math.abs(Math.abs(num) - 0));
        }

        System.out.println(mini);
        scanner.close();
    }
}

File: 800/C++/p11.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin>>n;
    
    int mini = INT_MAX;
    for(int i = 0; i < n; i++) {
        int num;
        cin>>num;
        mini = min(mini, abs(abs(num)-0));
    }

    cout<<mini<<endl;

    return 0;
}